tests/e2e-tests/specs/wp-admin/wp-admin-settings-tax.test.js

```diff
@@ -0,0 +1,175 @@
+		await activatePlugin( 'woocommerce' );
+	} );
+
+	it( 'can enable tax calculation', async() => {
+		// Go to general settings page
+		await StoreOwnerFlow.openSettings( 'general' );
+
+		// Make sure the general tab is active
+		await expect( page ).toMatchElement( 'a.nav-tab-active', { text: 'General' } );
+
+		// Enable tax calculation
+		await setCheckbox( 'input[name="woocommerce_calc_taxes"]' );
+		await settingsPageSaveChanges();
+
+		// Verify that settings have been saved
+		await Promise.all( [
+			expect( page ).toMatchElement( '#message', { text: 'Your settings have been saved.' } ),
+			verifyCheckboxIsSet( '#woocommerce_calc_taxes' ),
+		] );
+
+		// Verify that tax settings are now present
+		await expect( page ).toMatchElement( 'a.nav-tab', { text: 'Tax' } );
+	} );
+
+	it( 'can set tax options', async () => {
+		// Go to tax settings page
+		await StoreOwnerFlow.openSettings( 'tax' );
+
+		// Make sure the tax tab is active
+		await expect( page ).toMatchElement( 'a.nav-tab-active', { text: 'Tax' } );
+
+		// Prices exclusive of tax
+		await expect( page ).toClick( 'input[name="woocommerce_prices_include_tax"][value="no"]' );
+		// Tax based on customer shipping address
+		await expect( page ).toSelect( '#woocommerce_tax_based_on', 'Customer shipping address' );
+		// Standard tax class for shipping
+		await expect( page ).toSelect( '#woocommerce_shipping_tax_class', 'Standard' );
+		// Leave rounding unchecked (no-op)
+		// Display prices excluding tax
+		await expect( page ).toSelect( '#woocommerce_tax_display_shop', 'Excluding tax' );
+		// Display prices including tax in cart and at checkout
+		await expect( page ).toSelect( '#woocommerce_tax_display_cart', 'Including tax' );
+		// Display a single tax total
+		await expect( page ).toSelect( '#woocommerce_tax_total_display', 'As a single total' );
+
+		await settingsPageSaveChanges();
+
+		// Verify that settings have been saved
+		await Promise.all( [
+			expect( page ).toMatchElement( '#message', { text: 'Your settings have been saved.' } ),
+			verifyValueOfInputField( 'input[name="woocommerce_prices_include_tax"][value="no"]', 'no' ),
+			expect( page ).toMatchElement( '#woocommerce_tax_based_on', { text: 'Customer shipping address' } ),
+			expect( page ).toMatchElement( '#woocommerce_shipping_tax_class', { text: 'Standard' } ),
+			expect( page ).toMatchElement( '#woocommerce_tax_display_shop', { text: 'Excluding tax' } ),
+			expect( page ).toMatchElement( '#woocommerce_tax_display_cart', { text: 'Including tax' } ),
+			expect( page ).toMatchElement( '#woocommerce_tax_total_display', { text: 'As a single total' } ),
+		] );
+	} );
+
+	it( 'can add tax classes', async () => {
+		// Go to tax settings page
+		await StoreOwnerFlow.openSettings( 'tax' );
+
+		// Make sure the tax tab is active
+		await expect( page ).toMatchElement( 'a.nav-tab-active', { text: 'Tax' } );
+
+		// Remove additional tax classes
+		await clearAndFillInput( '#woocommerce_tax_classes', '' );
```

Comment:
Not sure if this rely works, since I'm getting this message:
```
 ● WooCommerce Tax Settings › can add tax classes

    expect(received).toBe(expected) // Object.is equality

    Expected: ""
    Received: "Reduced rate
    Zero rate"
```

So if there's any value, like the default ones, this test will fail, also make "WooCommerce Tax Settings › can set rate settings" fail as well, since there's no new tax class.

Reply:
@claudiosanches thanks for the review! I can reproduce the error you are having if I add tax classes before the test runs. 

I changed the way we verify that existing tax classes have indeed been deleted before creating a `Fancy` tax class and it is now working with existing tax classes. Please re-review. Thanks!